napari/plugins/io.py

```diff
@@ -208,6 +241,11 @@ def _write_multiple_layers_with_plugins(
     if plugin_name:
         # if plugin has been specified we just directly call napari_get_writer
         # with that plugin_name.
+        if plugin_name not in plugin_manager.plugins:
+            raise ValueError(
+                f"There is no registered plugin named {plugin_name!r}.\n"
+                f"Plugin names include: {set(plugin_manager.plugins)!r}"
```

Comment:
Include, or are? This looks like all available plugins? Also, should we only list the plugins that provide a writer hook spec here? ie “The available plugins for saving are:”

Reply:
fixed